common/sgx_workload/CMakeLists.txt

```diff
@@ -16,17 +16,25 @@ CMAKE_MINIMUM_REQUIRED(VERSION 3.2 FATAL_ERROR)
 
 INCLUDE(CMakeVariables.txt)
 
+SET(SGX_ENCLAVE_INCLUDE "${SGX_SDK}/include"
```

Comment:
Why is this needed ? SGX_ENCLAVE_INCLUDE is not getting used looks like. Below lines are already included. Or you can move SGX_ENCLAVE_INCLUDE in CMakeVariables.txt and use it here.
TARGET_INCLUDE_DIRECTORIES(${WORKLOAD_STATIC_NAME} PUBLIC "${SGX_SDK}/include/libcxx")
TARGET_INCLUDE_DIRECTORIES(${WORKLOAD_STATIC_NAME} PUBLIC "${SGX_SSL}/include")

Reply:
Removed unused variable.